woocommerce/woocommerce-paypal-payments
# Prevent displaying smart button multiple times on variable product page (2958)

### Description



Sometimes all variations of PayPal buttons appear on the product page.
![image](https://github.com/user-attachments/assets/a2616a9a-c395-422a-bc2c-68e9e546799d)


### Steps to Test




1. Set the subscription mode to "PayPal Subscriptions."
2. Create a variable product with three variations and link each variation to PayPal.
3. Navigate to the product page.
4. Add the product to the cart.
5. Observe that three PayPal smart buttons appear on the product page (**now fixed**)

![chrome_4giXaYRDMQ](https://github.com/user-attachments/assets/201bb7f2-d3d6-4fe7-91dd-cf121526259c)
